src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import Icon from "../Icons/Icon";
+import { IconNames } from "../Icons/IconTypes";
+
+export interface DropdownProps {
+  /** Inner label on the button that opens the dropdown */
+  dropdownLabel?: React.ReactNode;
+  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+  blockName?: string;
+  /** className you can add in addition to 'input' */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+  /** dropdownLabel's input ID */
```

Comment:
Since we already have the public `id`, so we need the additional `inputId` to be public?